actions/image_partition_action.go

```diff
@@ -12,6 +12,7 @@ Yaml syntax:
    imagename: image_name
    imagesize: size
    partitiontype: gpt
+   ptuuid: string
```

Comment:
```suggestion
   partuuid: string
```

Reply:
Thanks for your review and suggestion.
`ptuuid` means uuid  value for **partition table**
 `partuuid`  has been used for **partition** uuid variable in https://github.com/go-debos/debos/pull/297/commits/24b4a83cd978534305b9c6c43e268d196a14f390

Reply:
Yes, sure. I will revise it.